ts/src/conductor.ts

```diff
@@ -19,22 +19,18 @@ import { v4 as uuidv4 } from "uuid";
 import { _ALLOWED_ORIGIN } from "./conductor-helpers.js";
 import { makeLogger } from "./logger.js";
 import { AgentsAppsOptions, AppOptions } from "./types.js";
+import { readFileSync, writeFileSync } from "node:fs";
+import yaml from "js-yaml";
 
-const logger = makeLogger("Local Conductor");
-
-const HOST_URL = new URL("ws://localhost");
-const DEFAULT_TIMEOUT = 60000;
-const LAIR_PASSWORD = "lair-password";
+const logger = makeLogger();
```

Comment:
No need for context anymore with the logger?

Reply:
I think no. It'll default to a prefix `Tryorama` and I don't see the need to output "local conductor" anymore, as there are only local conductors now.